Antonio_Ramos_Documentation

```diff
@@ -140,12 +148,86 @@ new code - pFC = r"G:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin\CARR\California\Ba
+  "File "G:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin\kfunctions\Three Stage K-functions\kfunction_3Stage.py", line 49, in countSimulation
+    fsFeatures['FS_PROBS'] = fsFeatures[[fieldFS]] / float(sum(fsFeatures[fieldFS]))
+
+TypeError: unsupported operand type(s) for +: 'int' and 'str'"
+
+-(now working in kfunction_3Stage.py script) when running lines 42-49 (so as to include the definition of "fsFeatures") no error occurs and script runs perfectly fine. Moreover, when running the entire function definied as "countSimulation" (lines 42-87) no error is retunred and function executes. Similarly when running the entire function defined as "pointSimulation" (which is referenced in line 122 of K-function Local_3Stage.py) no error is returned and function executes. So I am confused as to what is sending us into this infinite loop. I believe it might have to do with the data we are feeding into the programs but I am not sure why. As mentioned above the attribute names being referenced coincide between the program and the data. 
+
+- Will mentions having come across a similar problem in his documentation on lines 143-145. I sent him an email asking if he remembers how he got around that problem, his solution is not articulated in the documentation. 
+
+06/11/2020
+- after communicating with Will, we were still not able to get documentation which aligns with the data referenced in the python scripts. Documentation that Will sent over was the same as what was already in the github repository. File not downloaded since it is the same as what is in the git repo. 
+
+- finished filling in Data & Script File Table which contains a list of all the files mentioned by Will Heikoff in his documentation, or utilized by the python script "K-function Local_3Stage.py" saved it in G:\MAX-Filler\Collab\Labs-kbuzard-S18\Admin . 
+
+- will now go through the following programs which Will Heikoff wrote to change aspects of the data so that it would work with the python scripts.
+1. ZIPS_Tot_Emp.r 
+2. CA_WAC_Transformation.R
+3. ZCTA_Manufa_Trans.R
+4. ZCTA_GoodLabs_Trans.py
+5. ZCTA_Manufa_Trans.py 
+6. GoodLabs_Trans.py
+(more complete descriptions avaliable in Data & Script File Table located at G:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin) 
+
+- discovered that the two of the three data files, GoodLabs_New.shp and CA_ZCTA_Man.shp, from the K-function Local_3Stage.py script come from one python script, "ZCTA_GoodLabs_Trans.py" (located at "G:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin\kfunctions\Three Stage K-functions\ZCTA_GoodLabs_Trans.py") 
+
+- ZCTA_GoodLabs_Trans.py changes the CRS for GoodLabs.shp and exports GoodLabs_New.shp 
```

Comment:
Is the only difference between GoodLabs.shp and GoodLabs_New.shp that the CRS is changed? If so, please note here.

Reply:
Yes that is the only difference I will note it